packages/salesforcedx-vscode-core/src/commands/stopApexDebugLogging.ts

```diff
@@ -30,7 +38,13 @@ export const turnOffLogging = async (extensionContext: vscode.ExtensionContext):
     extensionContext.workspaceState.update(userSpecificKey, undefined);
 
     disposeTraceFlagExpiration();
-    await handleFinishCommand(command, true);
+
+    channelService.showCommandWithTimestamp(
+      `${nls.localize(coerceMessageKey('long_command_end'))} ${nls.localize(coerceMessageKey(command))}`
+    );
+
+    await notificationService.showInformationMessage(`${nls.localize(coerceMessageKey(command))} successfully ran`);
+    telemetryService.sendCommandEvent(command);
   } else {
     await notificationService.showInformationMessage('No active trace flag found.');
   }
```

Comment:
I realized that in the current implementation, going to the else case would result in the user seeing a "Starting" message but no "Ended" message, which is confusing.  Since this line is needed in both the if and else blocks, I moved it outside.

```suggestion
  }
  await handleFinishCommand(command, true);
```